Src/VimCore/Interpreter_Parser.fs

```diff
@@ -1490,12 +1490,31 @@ type Parser
                 moveNextChar()
                 if afterEscape then
                     match c with
+                    | 'e' -> builder.AppendChar (char 0x1b)
                     | 't' -> builder.AppendChar '\t'
                     | 'b' -> builder.AppendChar '\b'
                     | 'f' -> builder.AppendChar '\f'
                     | 'n' -> builder.AppendChar '\n'
                     | 'r' -> builder.AppendChar '\r'
-                    | '\\' -> builder.AppendChar '\\'

```

Comment:
Why did this case get removed? Seems to be valid. Tried the following in gVim `:echo "f\\o"` and it printed `f\o`

Reply:
Because `\[char]` where `[char]` is not special already just appends that character. In other words, `\\` would already append `\` (by falling through to the wildcard match) so this line, while not harmful, is redundant.

In summary, `\` after `\` is not special, it's just another escaped character.

Reply:
I added a commit to specifically test that backslash escapes itself.